src/Raven.Server/Commercial/LicenseHelper.cs

```diff
@@ -374,15 +370,55 @@ public void AppendResolutionSuggestions()
                         _errorBuilder.AppendLine($"- As a temporary measure, consider downgrading to the last working build ({buildInfo.FullVersion}).");
                 }
 
-                AppendSuggestionToDisableThrowOnInvalidOrMissingLicenseOption(_configuration.Licensing.ThrowOnInvalidOrMissingLicense, _isInStorageLicenseExpired);
+                AppendSuggestionToDisableThrowOnInvalidOrMissingLicenseOption(Configuration.Licensing.ThrowOnInvalidOrMissingLicense, _isInStorageLicenseExpired);
             }
 
             public void AppendGeneralSuggestions()
             {
                 _errorBuilder.AppendLine();
                 _errorBuilder.AppendLine("To resolve this issue, you may consider the following options:");
-                _errorBuilder.AppendLine("- Ensure your license key is correctly embedded in 'settings.json', set as an environment variable, or included in your 'ServerOptions' if using an embedded server or Raven.TestDriver.");
-                _errorBuilder.AppendLine("- Alternatively, check the 'License.Path' in your configuration to ensure it points to a valid 'license.json' file.");
+                _errorBuilder.AppendLine("- Ensure your license key is correctly embedded in 'settings.json' or set as an environment variable.");
+                _errorBuilder.AppendLine("- Or, check the 'License.Path' in your configuration to ensure it points to a valid 'license.json' file.");
```

Comment:
instead of 'License.Path' use RavenConfiguration.GetKey method

Reply:
Done